### PR TITLE
feat(tracing): Allow multiple average columns

### PR DESCRIPTION
**Pull request by wmak**
- This updates the averageColumn param to be a list so multiple columns can be passed, so that the frontend can load both span.self_time and span.duration

### CODE REVIEW

**Review by Abdkhan14 — APPROVED** [review]

File: src/sentry/snuba/referrer.py [hunk 1]
```diff
@@ -349,6 +349,7 @@ class Referrer(Enum):
     )
     API_PERFORMANCE_VITAL_DETAIL = "api.performance.vital-detail"
     API_PERFORMANCE_VITALS_CARDS = "api.performance.vitals-cards"
+    API_PERFORMANCE_ORG_EVENT_AVERAGE_SPAN = "ape.performance.org-event-average-span"
```
Comment (narsaynorath) [inline src/sentry/snuba/referrer.py:352]:
```suggestion
    API_PERFORMANCE_ORG_EVENT_AVERAGE_SPAN = "api.performance.org-event-average-span"
```

File: tests/snuba/api/endpoints/test_organization_event_details.py [hunk 2]
```diff
@@ -325,7 +326,53 @@ def test_get(self):
...
+        for entry in entries:
+            if entry["type"] == "spans":
+                for span in entry["data"]:
+                    if span["op"] == "db":
+                        assert span[self.RESULT_COLUMN] == {"avg(span.self_time)": 1.0}
+                    if span["op"] == "django.middlewares":
+                        assert self.RESULT_COLUMN not in span
+
+    def test_invalid_column(self):
+        # If there's nothing stored for a metric, span.duration in this case the query returns nan
```
Comment (narsaynorath) [inline tests/snuba/api/endpoints/test_organization_event_details.py:369]:
```suggestion
        # If any columns are invalid, ignore average field in results completely
```

**Review by narsaynorath — APPROVED** [review]
Some small comments but lgtm

File: src/sentry/snuba/referrer.py (hunk 1 above)
Comment (0Calories) [reply src/sentry/snuba/referrer.py:352]: 🦍

Comment (wmak) [reply src/sentry/snuba/referrer.py:352]: OUG

**Pull request merged**